tests/src/proxy/test_locomotion.cpp

```diff
@@ -16,26 +16,51 @@ int main(int argc, char* argv[]) {
     proxy::Locomotion locomotion{locomotion_config};
 
     TestCore::loop([&button, &locomotion]() {
-        while (button.get_status() == proxy::Button::Status::NO_PRESS) { }
-
-        locomotion.set_wheel_command(50.0F, 0.0F);
-        hal::Timer::sleep_ms(1000);
-        locomotion.set_wheel_command(-50.0F, 0.0F);
-        hal::Timer::sleep_ms(1000);
-        locomotion.set_wheel_command(0.0F, 50.0F);
-        hal::Timer::sleep_ms(1000);
-        locomotion.set_wheel_command(0.0F, -50.0F);
-        hal::Timer::sleep_ms(1000);
-
-        locomotion.set_command(50.0F, 0.0F);
-        hal::Timer::sleep_ms(1000);
-        locomotion.set_command(-50.0F, 0.0F);
-        hal::Timer::sleep_ms(1000);
-        locomotion.set_command(0.0F, 50.0F);
-        hal::Timer::sleep_ms(1000);
-        locomotion.set_command(0.0F, -50.0F);
-        hal::Timer::sleep_ms(1000);
-        locomotion.set_command(0.0F, 0.0F);
+        auto button_status = button.get_status();
+
+        switch (button_status) {
+            case proxy::Button::Status::SHORT_PRESS:
+                locomotion.set_wheel_command(50.0F, 0.0F);
```

Comment:
diria que seria bacana essas velocidades serem `constexpr`, mudar isso em teste vai ser uma desgraça afiuhasuiffh principalmente qnd for testar coisas como rampa de aceleracao e os krl

Reply:
os tempos tambem